statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -626,6 +802,12 @@ def __init__(self, model):
         # Setup the cache
         self._cache = resettable_cache()
 
+        # Setup covariance matrix notes dictionary
+        if not hasattr(self, 'cov_kwds'):
+            self.cov_kwds = {}
+        self.cov_type = 'opg'
```

Comment:
Can you follow the pattern in the base and other classes and just overwrite `get_robustcov_results` with whichever covtypes you have in these classes? Then there wouldn't be any confusion with the inherited structure.
(I will open a seperate issue to check how the checks for cov_type in LikelihoodModelResults affects subclasses. There could possibly also problems with other subclasses that define their own cov types, and don't follow the same pattern.)

I thought `self.cov_type = 'opg'` was hardcoding the cov_type